proto/zitadel/settings/v2beta/settings_service.proto

```diff
@@ -395,12 +432,15 @@ service SettingsService {
 
   // Delete Organization Settings
   //
+  // Deprecated: please move to the corresponding endpoint under settings service v2. This endpoint will be removed with the next major version of ZITADEL.
```

Comment:
Do we need to change the defined deprecation notice in the [API_DESIGN](https://github.com/zitadel/zitadel/blob/main/API_DESIGN.md#deprecations)?

Reply:
I'll anyway create a separate PR to update it to ensure it's accurate again. Will include it there.